inc/osvr/ClientKit/Imaging.h

```diff
@@ -94,29 +92,25 @@ namespace clientkit {
                                      const struct OSVR_ImagingReport *report) {
                 ImagingCallbackRegistration *self =
                     static_cast<ImagingCallbackRegistration *>(userdata);
-                ImagingReportOpenCV newReport;
+                ImagingReport newReport;
                 newReport.sensor = report->sensor;
                 newReport.buffer.reset(report->state.data,
                                        ImagingDeleter(self->m_ctx));
-                newReport.frame = cv::Mat(
```

Comment:
Exactly, but don't we already have that opencv layer? `OpenCVTypeDispatch.h` is all a developer would need to wrap the buffer pointer into a matrix using the `cv::Mat` constructor, it would basically be a one-liner.